CONTRIBUTING.md

```diff
@@ -147,7 +147,8 @@ of the settings.
 
 If you are on Windows, you need to install **make** for the above process to complete.
```

Comment:
Do we have more instructions for people using Windoze?  Don't they need to be running cygwin or something and install make in that or is there something like a make.exe for Windoze?  It seems like this instruction may be too terse, given that this may be used by writers who are not super technical.

Reply:
yes valid point, but not part of this PR - i did not change content, i only changed the style to be compliant with markdown lint

we might need to add more information here, eg. where to find make for windows and a link to a how to guide, to set it up